clients/src/main/java/org/apache/kafka/clients/consumer/internals/OffsetsForLeaderEpochClient.java

```diff
@@ -85,10 +85,6 @@ protected OffsetForEpochResult handleResponse(
                 case KAFKA_STORAGE_ERROR:
                 case OFFSET_NOT_AVAILABLE:
                 case LEADER_NOT_AVAILABLE:
-                    logger().debug("Attempt to fetch offsets for partition {} failed due to {}, retrying.",
```

Comment:
Side cleanup

Reply:
Why we can remove this logic?

Reply:
It is exactly the same as the subsequent cases handling.